packages/api/src/constants.ts

```diff
@@ -0,0 +1,30 @@
+/**********************************************************************
```

Comment:
I would avoid to pickup a name "constant" as from an external POV I don't really know what it's being provided

it's like if I would name all utility classes "utility.ts" or every classes "class.ts"

Reply:
it contains constants that are related to the repository owner/name